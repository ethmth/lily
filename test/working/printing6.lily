let a: int = 5


print(a / 2)
print(6 / 2)
print(5.0 / 2.0)

if (! (4  == 4)):
    print(True)