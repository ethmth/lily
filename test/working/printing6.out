2
3
2.500000
